CMakeLists.txt

```diff
@@ -168,21 +168,38 @@ set(FlatBuffers_Library_SRCS
 
 set(FlatBuffers_Compiler_SRCS
   ${FlatBuffers_Library_SRCS}
+  src/idl_gen_binary.h
```

Comment:
Is it necessary to add header files to `FlatBuffers_Compiler_SRCS` here

Reply:
Removed.